cost-analyzer/templates/cost-analyzer-cluster-role-template.yaml

```diff
@@ -29,13 +29,13 @@ rules:
       - ''
     resources:
       - configmaps
-      - deployments
       - nodes
       - pods
       - events
       - services
       - resourcequotas
       - replicationcontrollers
+      - replicasets
```

Comment:
@chipzoller I think replicasets is in apps? https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#replicaset-v1-apps

Reply:
Dang it...